services/core-api/app/api/mines/mine/resources/mine.py

```diff
@@ -293,15 +317,22 @@ class MineResource(Resource, UserMixin):
         'exemption_fee_status_code',
         type=str,
         help='Fee exemption status for the mine.',
-        trim=True,
         store_missing=False,
+        trim=True,
         location='json')
     parser.add_argument(
         'exemption_fee_status_note',
```

Comment:
This didn't already exist on the mine resource? interesting

Reply:
nope, and it was ignoring those values on post